tools/scripts/renode-test-update.sh

```diff
@@ -20,6 +20,9 @@ quit_renode() {
 
 rm -f $RENODE_UART
 
+#
+# LMS and ext_LMS
+#
 if (echo $TEST_OPTIONS | grep "LMS" &>/dev/null); then
     # Need git.
     apt install -y git
```

Comment:
This is now useless, we just want to follow the "latest merged" version in this test so all the test results having the same codebase.
Please remove

Reply:
It's cleaned up now. Only does minimal steps to add & prep the external libs.